Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6-alpine
+
+RUN apk add --update --no-cache gcc g++ && pip install dumb-init
+RUN apk add --update --no-cache libxml2-dev libxslt-dev git
+
+VOLUME /var/lib/sirbot
```

Comment:
Also for layers, I think you can collapse these together: `VOLUME /var/lib/sirbot /var/log/sirbot`